### PR TITLE
Add psp-util plugin

### PR DESCRIPTION
**Pull request by jlandowner**
This PR adds the [psp-util](https://github.com/jlandowner/psp-util) plugin to the krew index. 

This plugin is a utility to manage Pod Security Policy(PSP) and related RBAC Resources.
Attach/Detach PSP to/from RBACs(Group, User) or ServiceAccounts and view the relations which PSP is effected to the Subjects in cluster.

- [x] Make sure you read the Plugin Naming Guide: https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/

I think it's ok but please review.

- [x] Verify you can install your plugin locally: kubectl krew install --manifest=[...] --archive=[...]

Locally verified on darwin_amd64 and linux_amd64.
```shell
$ k krew install --manifest ./plugins/psp-util.yaml 
Installing plugin: psp-util
Installed plugin: psp-util
\
 | Use this plugin:
 |      kubectl psp-util
 | Documentation:
 |      https://github.com/jlandowner/psp-util
/
WARNING: You installed plugin "psp-util" from the krew-index plugin repository.
   These plugins are not audited for security by the Krew maintainers.
   Run them at your own risk.
```

### CODE REVIEW

File: plugins/psp-util.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  creationTimestamp: null
```
Comment (ahmetb) [inline plugins/psp-util.yaml:4]: might be wise to remove this line

File: plugins/psp-util.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  name: psp-util
+spec:
+  description: |
+    A Kubectl plugin to manage Pod Security Policy(PSP) and the related RBAC Resources.
+    Attach/Detach PSP to/from RBACs(Group, User) or ServiceAccounts and
+    view the relations which PSP is effected to the Subjects in cluster.
+  homepage: https://github.com/jlandowner/psp-util
+  platforms:
```
Comment (ahmetb) [inline plugins/psp-util.yaml:12]: I'm assuming you can probably also provide a windows build since it's written in go. No action needed for this PR.

**Comment by ahmetb** [comment]
I see [in your code](https://github.com/jlandowner/psp-util/blob/1363c691c7c333f93ff698d3b3ee2d82e14d9f1f/pkg/client/client.go#L24) you only import gcp auth plugin:

    _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"


that might be issue for some users on other providers. If you import `/auth` directly, it'll be better.

Other than that, I don't have much to add.
/approve

**Comment by jlandowner** [comment]
> I see [in your code](https://github.com/jlandowner/psp-util/blob/1363c691c7c333f93ff698d3b3ee2d82e14d9f1f/pkg/client/client.go#L24) you only import gcp auth plugin:
> 
> ```
> _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
> ```
> 
> that might be issue for some users on other providers. If you import `/auth` directly, it'll be better.

Thank you for your review even my code!
I will fix it and update version to v1.0.1.

File: plugins/psp-util.yaml (hunk 1 above)
Comment (jlandowner) [reply plugins/psp-util.yaml:4]: I will fix it.

File: plugins/psp-util.yaml (hunk 2 above)
Comment (jlandowner) [reply plugins/psp-util.yaml:12]: Thanks. I will provide a windows build in next version.

**Comment by ahmetb** [comment]
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

/lgtm
/approve

**Comment by jlandowner** [comment]
I appreciate your help!

**Pull request merged**